src/containers/AssetList.js

```diff
@@ -5,6 +5,7 @@ import AssetTable from '../components/AssetTable';
 import Page from '../containers/Page';
 import GetMoreAssets from '../components/GetMoreAssets';
 import { withRouter } from 'react-router'
+import { withStyles } from 'material-ui/styles';
 import { connect } from 'react-redux';
 import { getAssets, Direction } from '../redux/actions/assetRegisterApi';
 
```

Comment:
We can remove the css import here if removing the last class.

Reply:
Not shown in the preview above, but I'm referring to: `import '../style/App.css';`

Reply:
wilco